### ISSUE TITLE
Error: Couldn't run ps ax: {}

### ISSUE THREAD

**Issue by bodiam** [meteor/meteor#2158]
Hi all,

I just downloaded the newest version of Meteor, and when I try to start a simple application on my Mac, I get the following error: Error: Couldn't run ps ax: {}

```
LM-AMS-00963649:my_cool_ap2 epragt$ meteor
[[[[[ ~/Projects/playground/my_cool_ap2 ]]]]]

=> Started proxy.
=> Starting MongoDB... -
/Users/epragt/.meteor/tools/6f23056589/lib/node_modules/fibers/future.js:206
                        throw(ex);
                              ^
Error: Couldn't run ps ax: {}
    at Object.Future.wait (/Users/epragt/.meteor/tools/6f23056589/lib/node_modules/fibers/future.js:326:15)
    at findMongoPids (/Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:84:14)
    at findMongoAndKillItDead (/Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:114:14)
    at launchOneMongoAndWaitForReadyForInitiate (/Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:229:5)
    at launchMongo (/Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:491:7)
    at _.extend._startOrRestart (/Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:580:19)
    at _.extend.start (/Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:546:10)
    at _.extend.start (/Users/epragt/.meteor/tools/6f23056589/tools/run-all.js:131:24)
    at Object.exports.run (/Users/epragt/.meteor/tools/6f23056589/tools/run-all.js:272:10)
    at main.registerCommand.name [as func] (/Users/epragt/.meteor/tools/6f23056589/tools/commands.js:210:17)
    at /Users/epragt/.meteor/tools/6f23056589/tools/main.js:949:23
    - - - - -
    at /Users/epragt/.meteor/tools/6f23056589/tools/run-mongo.js:55:22
    at ChildProcess.exithandler (child_process.js:633:7)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:743:16)
    at Process.ChildProcess._handle.onexit (child_process.js:810:5)
LM-AMS-00963649:my_cool_ap2 epragt$
```

**Comment by bigtbigtbigt** [comment]
I was experiencing this with Meteor 0.8.1.3, and I solved the issue by closing several of my many open applications.

**Comment by glasser** [comment]
This was fixed in Meteor 0.8.2 last June, for the record.
